include/wx/osx/textentry.h

```diff
@@ -72,6 +72,11 @@ class WXDLLIMPEXP_CORE wxTextEntry: public wxTextEntryBase
     virtual bool CanUndo() const wxOVERRIDE;
     virtual bool CanRedo() const wxOVERRIDE;
 
+    virtual void EmptyUndoBuffer();
```

Comment:
```suggestion
    virtual void EmptyUndoBuffer() wxOVERRIDE;
```
to avoid clang warnings about inconsistent use of `override`.